docs/release-notes.md

```diff
@@ -1,8 +1,37 @@
+## 0.19.0
+
+March, 9, 2022
```

Comment:
Sorry to be too picky but I think the pattern is `March 9, 2022`?

Reply:
```suggestion
March 9, 2022
```